_docs/README.md

```diff
@@ -5,9 +5,9 @@ Ubiquitous is a multi-purpose Moodle hosting environment.
 It fulfils three requirements:
 
 1. _Provisioning_ of machines with either a container or virtualisation platform.
-    * In local development environments we manage [VirtualBox](https://www.virtualbox.org/) virtual machines with [Vagrant](https://www.vagrantup.com/)
+    * In local development environments we manage [VirtualBox](https://www.virtualbox.org/) virtual machines with [Vagrant](https://www.vagrantup.com/) (version [1.9.6](https://releases.hashicorp.com/vagrant/1.9.6/) for now, because of [this bug](https://github.com/mitchellh/vagrant/issues/8770)).
     * Continuous integration platforms run on purpose-built [Docker](https://www.docker.com/) containers
```

Comment:
Finally, someone who cares about this! Here's how I view list formatting:
- if list elements are not full sentences (i.e. they lack predicates), only the last of them gets a period at the end, since the list is pretty much like a standard comma-based one and its enclosing sentence has begun just above the list.
- if list elements contain predicates, each gets a period, since they are full sentences.
-  we should avoid mixed lists, i.e. lists that mix both of the above.
The latest commit uses these rules - see if you like how it looks.